tests/integration/conftest.py

```diff
@@ -368,19 +368,36 @@ def repositories(shell):
 
 
 @pytest.fixture(autouse=True)
-def missing_centos_release_workaround(system_release, shell):
+def missing_os_release_package_workaround(shell):
     # TODO(danmyway) remove when/if the issue gets fixed
     """
-    Fixture to workaround issues with missing `centos-linux-release`
-    after incomplete rollback.
+    Fixture to workaround issues with missing `*-linux-release`
+    package, after incomplete rollback.
     """
     # run only after the test finishes
     yield
 
-    if "centos-8.5" in system_release:
-        rpm_output = shell("rpm -q centos-linux-release").output
+    os_to_pkg_mapping = {
+        "oracle-7": "oracle-release-el7",
+        "oracle-8": "oraclelinux-release-el8",
+        "centos-7": "centos-release",
+        "centos-8": "centos-linux-release",
```

Comment:
Ideally we should keep it to centoslinux-8 so that we could support stream. But this is fine for now, we can change later when stream is introduced